src/sentry/static/sentry/app/views/organizationStream/overview.jsx

```diff
@@ -104,6 +105,24 @@ const OrganizationStream = createReactClass({
     }
   },
 
+  componentWillReceiveProps(nextProps) {
```

Comment:
Is there anyway we can run this block of code in `componentDidMount` instead to keep it together with the rest of the refetching code. `componentWillReceiveProps` is getting deprecated.